corehq/apps/users/util.py

```diff
@@ -126,18 +126,12 @@ def user_id_to_username(user_id, use_name_if_available=False):
         return raw_username(user_object['username']) if "username" in user_object else None
 
 
+@quickcache(['user_id'])
 def cached_user_id_to_username(user_id):
-    if not user_id:
+    if not user_id or user_id == 'system':
```

Comment:
Nitpick: there are constants for this, either `SYSTEM_USER_TYPE` or `SYSTEM_USER_ID`